guides/Pagination.md

```diff
@@ -61,26 +87,79 @@ include this structure in the request body:
   }
 }
 ```
+:::
+:::SDK_TAB
+import { contacts } from 'wix-crm-backend';
+
+contacts.query()       
+  .ascending('info.name.last')                 // Sort results by last name (A–Z)
+  .descending('createdDate')                   // Then by creation date (newest first)
+  .find()
+  .then(results => {
+    console.log(results.items); // Sorted and filtered results
+  })
+  .catch(error => {
+    console.error(error);
+  });
+:::
+::::
 
 ## Paging
 
 The standard Wix API pagination includes:
 
 - **limit**: amount of items per response (default is `0`)
 
-- **offset**: number of items to skip
+- **offset/skip**: number of items to skip. The standard in REST is **offset**, and in the SDK is **skip**.
 
-The following examples:
+- **cursor paging**: 
 
+### Paging _List_ endpoints
+
+For example, to list 100 contacts, starting from contact 20:
+
+::::tabs
+:::REST_TAB
 ```txt
 ?limit=100&offset=20
 ```
+:::
+:::SDK_TAB
```

Comment:
full code sample:
import { listContacts } from 'wix-crm-backend';

const sort = [
  { fieldName: "info.name.last", order: "ASC" },   
  { fieldName: "createdDate", order: "DESC" }      
];

listContacts({
  sort,     
  limit: 100, // Number of items to return
  skip: 20  // Number of items to skip (i.e., start with contact #101)
})
.then(results => {
  console.log(results.items);
})
.catch(error => {
  console.error(error);
});

Reply:
SDK list method with offset paging: https://dev.wix.com/docs/sdk/backend-modules/members/members/list-members